internal/httpserve/handlers/ent.go

```diff
@@ -501,3 +500,86 @@ func (h *Handler) getOrgByID(ctx context.Context, id string) (*ent.Organization,
 
 	return org, nil
 }
+
+// getOrgSettingByOrgID returns the organization settings from an organization ID and context
+func (h *Handler) getOrgSettingByOrgID(ctx context.Context, orgID string) (*ent.OrganizationSetting, error) {
+	orgGetCtx := privacy.DecisionContext(ctx, privacy.Allow)
```

Comment:
Why do we need to override and add `allow` here? This is called from an authorized handler, right?

If this is something we need a comment should be added about why.